desktop/src/navigator.rs

```diff
@@ -113,14 +106,33 @@ impl NavigatorBackend for ExternalNavigatorBackend {
     }
 
     fn fetch(&self, url: &str, _options: RequestOptions) -> OwnedFuture<Vec<u8>, Error> {
-        // Load from local filesystem.
-        // TODO: Support network loads, honor sandbox type (local-with-filesystem, local-with-network, remote, ...)
-        let mut path = self.relative_base_path.clone();
-        path.push(url);
-
-        Box::pin(async move { fs::read(path).map_err(Error::NetworkError) })
+        // TODO: honor sandbox type (local-with-filesystem, local-with-network, remote, ...)
+        let full_url = self.movie_url.clone().join(url).unwrap();
+
+        match full_url.scheme() {
+            "file" => Box::pin(async move { fs::read(full_url.to_file_path().unwrap()).map_err(Error::NetworkError) }),
+            _ => {
+                let mut client_builder = reqwest::Client::builder();
+                if let Some(proxy) = self.http_proxy.to_owned() {
+                    client_builder = client_builder.proxy(reqwest::Proxy::http(proxy.as_str()).unwrap())
+                }
+                if let Some(proxy) = self.https_proxy.to_owned() {
+                    client_builder = client_builder.proxy(reqwest::Proxy::https(proxy.as_str()).unwrap())
+                }
+                let client = client_builder.build().unwrap();
+                Box::pin(async move {
+                    get_response_bytes(client, full_url).map_err(|err| {
+                        match err.url() {
+                            Some(url) => Error::FetchError(url.as_ref().to_string()),
+                            None => Error::FetchError("'Bad URL'".to_string())
+                        }
+                    } )
```

Comment:
```
get_response_bytes(client, full_url)
                        .await
                        .map_err(|err| match err.url() {
                            Some(url) => Error::FetchError(url.to_string()),
                            None => Error::FetchError("Bad URL".to_string()),
                        })
```

Reply:
Due to the way the tokio function returns, an await would cause a compilation error here. Will clean up the style though.